### DISCUSSION TITLE
Videos download without thumbnails. Can't see thumbnails of videos in Windows Explorer

### DISCUSSION THREAD

**Discussion by dare25GH** [aclap-dev/video-downloadhelper#1175] [Help and Q&A] [answered]
### Your question

Hello, 

I was downloading videos via downloadhelper and the videos downloaded perfectly as expected along with the video thumbnail. Today however, the videos download, but without the thumbnails. Trying to add thumbnails manually via apps like mp3tag or tageditor is futile. The result is the same, be it on youtube, vimeo, patreon etc

Things I have tried :
Uninstall and reinstall downloadhelper.
Tried the extension on both firefox and chrome, having the same result.
Deleted cache from browsers and tried again.

Using firefox 129.0.2 (64-bit)
Win 10, 64 bit

Please fix this at the earliest as I only have 3 days access to the videos.
Thank you for your kind help.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 2,
    "successfull_dl": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

In my experience, the file manager in Windows 11 is a lot different from the one in Windows 7.  W7 used to show the thumbnails by default.  W11 hides them by default.

![#01](https://github.com/user-attachments/assets/27dbe66d-e62c-4365-8e0a-14497b89af0d)
![#02](https://github.com/user-attachments/assets/7f11ca57-535d-447f-8987-6ae6b945a0fe)

This is a Windows issue, not a VDH issue.  The thumbnails are there.  You just have to go out of your way to see them.  Personally, I think the way W11 handles thumbnails is horrible.  That is a huge amount of screen real estate taken up by stuff that W7 used to show so much more concisely.  This seems to be almost a fetish of Microsoft, to complicate things that used to be simple.  Or should I call them Bloatosoft.  If I want to see a thumbnail, I click that Details control, look at what it tells me, then click it again to revert to the default display.  I do not show thumbnails on my system by default.

Now, you did say you are running W10.  I skipped W10 & went straight from W7 to W11, so I have no idea how similar or different W10 & W11 are.  But I'm assuming they're similar enough that my images above from W11 are relevant to W10.

**Reply by GrampaWildWilly** [reply]
Here's an image I fished out of something I posted on our old Google Groups forum.  Back then, I was running W7.

![#15](https://github.com/user-attachments/assets/4a4bc2cb-80aa-4bac-a465-ad993c126fe3)

Look at the bottom area of the file manager window in that image.  All the useful information used to be right there, including the thumbnail.  No more.

Here's an even better example.

![#02](https://github.com/user-attachments/assets/f7dc86cc-30b7-495e-bbb4-2a43840342b9)

The area at the bottom of the file manager window could be expanded.  This second image shows the expanded information that used to be available.  It included the thumbnail.  But all that's changed now, & I say for the worse.

**Reply by dare25GH** [reply]
Hey GrampaWildWilly,

Thank you for your help! 

The icons are in the 'large icons' size in the explorer. Sharing a screenshot for your reference.

![dwhelper thumbnail problem ](https://github.com/user-attachments/assets/9289a5ca-ce6f-45d8-a645-77e2072bd5d8)

As you can see, the videos downloaded from the day before have their thumbnails visible. No software or changes were made in the OS yesterday.

Using the 'Preview' and 'Details' panes results in the same thumbnail-less previews

**Comment by dare25GH** [comment, marked as answer]
Was able to resolve the issue by installing K-lite codec pack.

**Reply by GrampaWildWilly** [reply]
Oh yeah.  Forgot all about that.  I have that, too.  It does make a big difference.
